.github/workflows/ci.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           name: stats
           path: entrypoint-sizes.json
           if-no-files-found: error
+      - name: Export the test coverage report
```

Comment:
That's a good point. I guess we could run the testsuite only once and have it be instrumented using `bisect_ppx`, but I am a little scared to do that, because then we'd rely on the correctness of `bisect_ppx` more than I'd like—it does alter the code after. If you are not worried about this at all, we could indeed make `nix-build -A michelson --arg doCheck true` always instrument with `bisect_ppx`.